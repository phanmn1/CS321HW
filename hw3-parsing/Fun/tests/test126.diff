1c1
< Invalid syntax at (8,3), x
---
> Invalid syntax at (14,1), n
