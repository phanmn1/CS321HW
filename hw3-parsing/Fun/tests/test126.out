Invalid syntax at (14,1), n
